src/rp/util/number_spec.clj

```diff
@@ -0,0 +1,6 @@
+(ns rp.util.number-spec
+  (:require [rp.util.number :as util-number]
+            [clojure.spec :as s]))
```

Comment:
I think we'd need to move these specs to a namespace in the `test` directory so projects using `rp.util` aren't dependent on Clojure 1.9.